x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list_page.test.tsx

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import * as reactTestingLibrary from '@testing-library/react';
+import { waitForEuiPopoverOpen } from '@elastic/eui/lib/test/rtl';
+import userEvent from '@testing-library/user-event';
+import {
+  type AppContextTestRender,
+  createAppRootMockRenderer,
+} from '../../../../common/mock/endpoint';
+import { ResponseActionsListPage } from './response_actions_list_page';
+import type { ActionListApiResponse } from '../../../../../common/endpoint/types';
+import { MANAGEMENT_PATH } from '../../../../../common/constants';
+import { getActionListMock } from '../../../components/endpoint_response_actions_list/mocks';
+
+let mockUseGetEndpointActionList: {
+  isFetched?: boolean;
+  isFetching?: boolean;
+  error?: null;
+  data?: ActionListApiResponse;
+  refetch: () => unknown;
+};
+jest.mock('../../../hooks/endpoint/use_get_endpoint_action_list', () => {
+  const original = jest.requireActual('../../../hooks/endpoint/use_get_endpoint_action_list');
+  return {
+    ...original,
+    useGetEndpointActionList: () => mockUseGetEndpointActionList,
+  };
+});
+
+jest.mock('@kbn/kibana-react-plugin/public', () => {
```

Comment:
Do you need to mock the entire module? I think its already all mocked up because we use the mocked core context - if that is true, could you just use  `useKibana().services.uiSettings.get.mockImplementaiton()` instead?

Reply:
Yeah, the uiSettings need to be mocked. This is also how it is done in other tests that use a date picker. You can see the test for the date picker on the Overview page.